src/main/java/org/minions/devfund/carlosorellana/Squarelotron.java

```diff
@@ -0,0 +1,204 @@
+
+                if ((i == first || i == last) && (j >= first && j <= last)) {
+                        squarelotron.squarelotron[j][i] = this.squarelotron[i][j];
+                }
+
+                if ((i > first && i < last) && (j == first || j == last)) {
+                        squarelotron.squarelotron[j][i] = this.squarelotron[i][j];
+                }
+            }
+        }
+        return squarelotron;
+    }
+
+    /**
+     * Rotates Matrix 90 grades positive or negative position.
+     *
+     * @param numberOfTurns Number or rotations.
+     */
+    public void rotateRight(int numberOfTurns) {
+
+        int ring;
+        boolean right = true;
+        int maxRing = getMaxRings();
+
+        // New Temporal Squarelotron
+        int[][] temporalSquarelotron = new int[size][size];
+
+        // Duplicates the squarelotron matrix.
+        for (int i = 0; i < size; i++) {
+            for (int j = 0; j < size; j++) {
+                temporalSquarelotron[i][j] = this.squarelotron[i][j];
+            }
+        }
+
+        if (numberOfTurns < 0) {
+            right = false;
+            numberOfTurns = numberOfTurns * (-1);
+        }
+
+        for (int x = 0; x < numberOfTurns; x++) { // Move the ring in base the number of turns.
+            for (int y = 1; y <= maxRing; y++) { // Mode on the ring selected
+                ring = y;
+                int first = ring - 1;
+                int last = size - ring;
+
+                for (int i = 0; i < size; i++) {
+                    for (int j = 0; j < size; j++) {
+
+                        if (i == first) {
+                            if (j >= first && j <= last) {
+                                this.squarelotron[i][j] = (right) ? temporalSquarelotron[size - 1 - j][first]
+                                        : temporalSquarelotron[j][last];
+                            }
+                        } else if (i == last) {
+                            if (j >= first && j <= last) {
+                                this.squarelotron[i][j] = (right) ? temporalSquarelotron[size - 1 - j][last]
+                                        : temporalSquarelotron[j][first];
+                            }
+                        } else if (i > first && i < last) {
+                            if (j == last) {
+
+                                this.squarelotron[i][j] = (right) ? temporalSquarelotron[first][i]
+                                        : temporalSquarelotron[last][size - 1 - i];
+                            } else if (j == first) {
+                                this.squarelotron[i][j] = (right) ? temporalSquarelotron[last][i]
+                                        : temporalSquarelotron[first][size - 1 - i];
+                            }
+                        }
+                    }
+                }
+            }
+
+//            // replace array to temporal array
+//            for (int i = 0; i < size; i++) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)